presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -975,6 +982,25 @@ public RowExpression rowExpression(String sql)
                 session);
     }
 
+    public RowExpression rowExpression(String sql, ParsingOptions.DecimalLiteralTreatment decimalLiteralTreatment)
+    {
```

Comment:
Is this related to this PR?

Reply:
The default parser option has DecimalLiteralTreatment default to reject, which will not parse the `0.1`, `0.2` etc. in the test SQL and throw exception. So I added another function call which enable DecimalLiteralTreatment to be `As_Double`.